Modules/Components/Elastix/test/selxElastixComponentTest.cxx

```diff
@@ -170,6 +176,8 @@ TEST_F( ElastixComponentTest, MonolithicElastixTransformix )
 
```

Comment:
These definitions of the Blueprint could also be done in a json file instead of in code.